gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_index.adoc

```diff
@@ -22,4 +22,6 @@ ifeval::["{task-prefix}" == "oc"]
 
 include::_openshift.adoc[]
 
+include::_vertx.adoc[]
```

Comment:
is it intended that it is in the oc doc only ?

Reply:
oh, nice catch! this looks wrong. Let me fix it.